.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: 'bug: unconfirmed'
+assignees: ''
+
+---
+
+- [] I have checked that this bug exists on the latest stable version of AutoGluon
+- [] and/or I have checked that this bug exists on the latest mainline of AutoGluon via source installation
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+A minimal script to reproduce the issue
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Installed Versions**
+Which version of AutoGluon are you are using?
+If you are using 0.4.0 and newer, or the current mainline:
+<details>
+
+# Replace this code with the output of the following:
+from autogluon.core.utils import show_versions
+show_versions()
```

Comment:
Add triple ` around this so it formats correctly as code:

"```"

We want users to put the output inside the "```" so it will format nicely (as you did in your example output in this GitHub Issue)